src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  let totalRentalCost = days * rentPerDay;
+
+  if (days >= 3) {
+    totalRentalCost = days * rentPerDay - 20;
+  }
+
+  if (days >= 7) {
+    totalRentalCost = days * rentPerDay - 50;
+  }
+
+  return totalRentalCost;
 }
```

Comment:
```suggestion
  const rentPerDay = 40;
  let totalRentalCost = days * rentPerDay;

  if (days >= 3) {
    totalRentalCost -= 20;
  }

  if (days >= 7) {
    totalRentalCost -= 50;
  }

  return totalRentalCost;
}
```

Reply:
be careful about the previous comment. you shouldn't make all variables constants, but you should move all numbers to constants with descriptive names

Reply:
if you have questions about it, please write in the chat